libs/application/templates/driving-license/src/forms/application.ts

```diff
@@ -132,6 +132,35 @@ export const drivingLicenseApplication: Form = buildForm({
         }),
       ],
     }),
+    buildSection({
+      id: 'healthDeclaration',
+      name: 'Heilbrigðisyfirlýsing',
+      children: [
+        buildMultiField({
+          id: 'healthDeclaration',
+          name: 'Heilbrigðisyfirlýsing',
+          children: [
+            buildIntroductionField({
+              id: 'intro',
+              name: '',
+              introduction:
+                'Ef sótt er um réttindi í flokkum <b>AM</b>, <b>A1</b>, <b>A2</b>, <b>A</b>, <b>B</b>, <b>BE</b> eða <b>T</b> nægir heilbrigðisyfirlýsing ein og sér, nema sýslumaður telji þörf á læknisvottorði eða ef umsækjandi hefur náð 65 ára aldri eða hann vilji heldur skila læknisvottorði. Með umsókn um aðra flokka ökuréttinda (aukin ökuréttindi) er krafist læknisvottorðs á sérstöku eyðublaði.',
```

Comment:
you can also use markdown if you want (see `introIntroduction` in `reference-template/src/forms/messages.ts` if you want an example)

Reply:
aaah nice!!!